KarhooUISDK/CommonUI/UIDimens.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  UIDimens.swift
+//  KarhooUISDK
+//
+//  Created by Diana Petrea on 05.01.2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+
+// Note: When updating the UI, aproximate the commented values to the ones before each comment. E.g.: replace any spacing of value 17 with .standard (which is 16)
+public struct UIDimens {
```

Comment:
`Avoid abbreviations` violation I guess :)

Reply:
Since `UIDimensions` is already used in UIKit maybe just `Dimensions` or `UIConstants`

Reply:
I like UIConstants better. I'll change it to that